README.md

```diff
@@ -29,6 +29,14 @@ In your `<body/>`:
         title: "Title",
         subtitle: "Subtitle",
         profileAvatar: "http://to.avat.ar",
+        params: {
+          images: {
+            dims: {
+              width: null,
+              height: null,
+            }
+          }
+        }
```

Comment:
May be provide an example with real numbers. I guess no one would really set those to null they would just not  dims